src/AspNet.Security.OpenId.Steam/SteamAuthenticationExtensions.cs

```diff
@@ -18,65 +18,45 @@ namespace Microsoft.AspNetCore.Builder
     public static class SteamAuthenticationExtensions
     {
         /// <summary>
-        /// Adds <see cref="SteamAuthenticationMiddleware"/> to the specified
+        /// Adds <see cref="SteamAuthenticationHandler"/> to the specified
         /// <see cref="IApplicationBuilder"/>, which enables Steam authentication capabilities.
         /// </summary>
-        /// <param name="app">The <see cref="IApplicationBuilder"/>.</param>
+        /// <param name="builder">The <see cref="IApplicationBuilder"/>.</param>
         /// <returns>The <see cref="IApplicationBuilder"/>.</returns>
-        public static IApplicationBuilder UseSteamAuthentication([NotNull] this IApplicationBuilder app)
-        {
-            return app.UseSteamAuthentication(new SteamAuthenticationOptions());
-        }
+        public static AuthenticationBuilder AddSteam([NotNull] this AuthenticationBuilder builder)
+            => builder.AddSteam(SteamAuthenticationDefaults.AuthenticationScheme, _ => { });
```

Comment:
Pretty sure I already mentioned that, but please bring back the null checks.

Reply:
[NotNull] check added